src/components/image-dropzone.js

```diff
@@ -19,7 +19,7 @@ const ImageDropZone = ({ handleImageDrop, image_url, name }) => {
   const alt = file ? file.name : name
 
   const thumb =
-    src.length > 0 ? (
+    src && src.length > 0 ? (
```

Comment:
Did this actually work for anything? If not, we should remove it for now and figure out the image stuff later

Reply:
it fixed buggy bois